src/databases/Tecplot/avtTecplotWriter.C

```diff
@@ -578,6 +584,9 @@ avtTecplotWriter::WriteUnstructuredMesh(vtkUnstructuredGrid *ug, int chunk)
           case VTK_TRIANGLE:
             ntri++;
             break;
+          case VTK_PIXEL:
+            // intentional fallthrough to VTK_QUAD because of identical behavior
+            // to avoid duplicating code
```

Comment:
Apologies...I think the fallthrough cases of (case: case: case: code...break;) are fine. The ones that are challenging, IMHO are the cases of (case: code... case:). In other words, if there is no code intervening `case xxx:` statements, that is fine. If there is code intervening `case xxx:` statements which do not also include a `break;` statement, then those cases I think require a _fallthrough_ comment.

So, the comment isn't needed here or in similar cases and IMHO, makes it a little less clean to have it there.

Does that make sense?